notebooks/classification/Cifar10.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
```

Comment:
The idea here is to have both `.ipynb` and `.md` files on github : the `.ipynb` file allows the user to directly visualize the tutorial with the figures while the `.md` file is used for code review.

Reply:
Hum, it is very painful to host `.ipynb` files on github, since any execution of the notebook will modify the date and contaminate any pull request. Can't we add a `markdowns` command in the makefile, with a simple `jupytext --to ipynb *.md` in it ?

Reply:
OK for publishing notebooks, but they cannot take over examples, and really hope they won't contaminate all pull requests